### ISSUE TITLE
Change loss function

### ISSUE THREAD

**Issue by terrie1028** [Dongwoo-Im/dacon_vqa#1]
Hi, @Dongwoo-Im.
First of all, thanks for sharing your code.

I have a question.
In the fine-tuning code, is it possible to replace the loss function with a loss other than that of the exsisting model?
If it is possible, I would appreciate it if you could tell me how.

Thank you!

**Comment by Dongwoo-Im** [comment]
@terrie1028 

Maybe you can change the loss function

You can track the loss value come from where (BlipTextLMHeadModel -> BlipForQuestionAnswering)

https://github.com/huggingface/transformers/blob/fd06ad5438249a055d0b2fd2fc2567d8265a7e4b/src/transformers/models/blip/modeling_blip.py#L1317-L1342

https://github.com/huggingface/transformers/blob/fd06ad5438249a055d0b2fd2fc2567d8265a7e4b/src/transformers/models/blip/modeling_blip_text.py#L893-L914

(transformer library version can be different)

**Comment by terrie1028** [comment]
Thank you for reply and sharing the reference!!
